src/modules/flight_mode_manager/FlightModeManager.hpp

```diff
@@ -64,6 +64,12 @@ enum class FlightTaskError : int {
 	ActivationFailed = -2
 };
 
+enum class Manual_Position_control_sub_mode : int32_t {
+	Simple_position_control = 0,
+	Smooth_position_control = 3,
+	Acceleration_based_input = 4,
+};
```

Comment:
Would be nice to keep with the code style guidelines please: http://docs.px4.io/main/en/contribute/code.html#code-style
```suggestion
enum class ManualPositionControlSubMode : int32_t {
	kSimplePositionControl = 0,
	kSmoothPositionControl = 3,
	kAccelerationBasedInput = 4,
};
```
Also .. maybe just being a bit pedantic, but any particular reason we need the `int32_t` specifier?